Apps/Admin/Client/Pages/DashboardPage.razor.cs

```diff
@@ -170,7 +178,7 @@ private int UniqueDays
 
             List<DailyDataRow> results = new();
 
-            if (this.RegisteredUsersResult?.Result != null)
+            if (this.RegisteredUsersResult?.Result != null && this.DashboardState.Value.RegisteredUsers.Loaded)
```

Comment:
Nit: these two conditions are checking the same thing; the `Loaded` getter just returns whether or not `Result` is null.

Reply:
Okay I didn't realize that. I thought it didn't work the original way (this.DependentsResult?.Result != null) but it was actually a bug in the reducer I had to fix. I got rid of the duplication.